build-support/scripts/up

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-: ${IMG_NAME:="nuodb/nuodb:latest"}
+: ${IMG_NAME:="nuodb/nuodb:5.1"}
```

Comment:
@eresende-nuodb already made the latest release work. Can you revert this change and see if the CircleCI build passes?

Reply:
I will change this back, since if probably make sense for overnights, I had left it that way since it kept failing for me.